lib/trino-collect/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>370-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-collect</artifactId>
+    <name>trino-collect</name>
```

Comment:
collect is not the best way. Maybe `trino-java-utils`?

Reply:
I don't want this to be a bag-for-anything. I am sure we can revisit the name itself, if needed to expand the scope.

Reply:
I'd prefer to scope this to `trino-cache`, since `collect` is more broad.